packages/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@labkey/components",
-  "version": "1.0.0",
+  "version": "1.0.0-20.11-fb-issue41460.2",
```

Comment:
Since this is a patch/fix for a LK release branch, we will need to use the "Making hofix patches to a release branch" process described here: https://github.com/LabKey/labkey-ui-components/blob/master/packages/components/docs/localDev.md#making-hotfix-patches-to-a-release-branch

Since there have been changes merged into the @labkey/components package since the 0.104.0 version, we'll want to make sure those don't get included in the new patch fix version.

Reply:
Rebased onto 0.104.0 version and retargeted merge to to `release20.11-SNAPSHOT`